packages/client/src/v2-events/features/events/components/Output.tsx

```diff
@@ -60,82 +66,76 @@ interface FieldWithValue {
  */
 function ValueOutput(field: FieldWithValue) {
   /* eslint-disable react/destructuring-assignment */
```

Comment:
I tried to do a solution which would look something like this:

```
  for (const fieldType of fieldTypes) {
    if (fieldType.isFieldType(field)) {
      return fieldType.Output({ value: field.value })
    }
  }

```

instead of all the if's below. But I was unable to get the type casting/guarding working correctly for field.value and field.config. If you have any ideas for this, let me know :)